vuvantien/chap2vvt/security/todo_access_rules.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <data noupdate="1">
+    <record id="todo_task_user_rule" model="ir.rule">
+      <field name="name">ToDo Tasks only for owner</field>
+      <field name="model_id" ref="model_todo_task"/>
+      <field name="domain_force">[('create_uid','=',user.id)]
+      </field>
+      <field name="groups" eval="[(4,ref('base.group_user'))]"/>
+    </record>
+  </data>
+</odoo>
```

Comment:
Đọc thêm trong sách xem có giải thích gì về đoạn này thì nói lại nhé (chụp ảnh đoạn text đưa lên comment của git)

Reply:
To write on a to-many field, we use a list of triples. Each triple is a write command that does different
things according to the code used:(0,_ ,{'field': value}) creates a new record and links it to this one
(1,id,{'field': value}) updates the values on an already linked record
(2,id,_) unlinks and deletes a related record
(3,id,_) unlinks but does not delete a related record
(4,id,_) links an already existing record
(5,_,_) unlinks but does not delete all the linked records
(6,_,[ids]) replaces the list of linked records with the provided list
The underscore symbol used in the preceding list represents irrelevant values, usually filled with 0 or
False .